runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -358,4 +358,26 @@ default boolean collapseEntries()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+			keyName = "inventoryStacks",
+			name = "Inventory Stacks",
+			description = "Highlights items already stacked in inventory",
+			position = 28
+	)
+	default boolean inventoryStacks()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+			keyName = "inventoryStacksColor",
+			name = "Stackable color.",
+			description = "Configures the highlight color for items already stacked in inventory",
+			position = 29
+	)
+	default Color inventoryStacksColor()
+	{
+		return Color.decode("#2AECFA");
```

Comment:
This should just use the existing "highlight" color instead of having its own highlight color imo.

Reply:
My thinking was that the existing highlight color was "sacred" in a way and reserved for items that you "really would want to" pick up, while this option is for items that you "might as well" pick up. I felt that different player behavior should present in different colors. 

With that said, this is my own opinion and maybe I am misunderstanding how the highlight list is meant to be used.

Should these stacks work like the normal highlight list in every other way as well? Right now, if the item is in the hidden list, but also in the player's inventory, the item is not highlighted. This is to avoid cluttering the screen with less valuable stackable drops the player might have in its inventory, such as coins or runes. The player also don't get desktop notifications for items that are already stacked in its inventory, should they?